src/pages/Categories/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { CategoryCard } from "../../components";
+
+const categories = [
+  {
+    title: "Tech",
+    imgSrc: "https://source.unsplash.com/FO7JIlwjOtU",
+    to: "/rules",
+  },
+  {
+    title: "Cricket",
+    imgSrc: "https://source.unsplash.com/IfO43eONJeo",
+    to: "/rules",
+  },
+  {
+    title: "India",
+    imgSrc: "https://source.unsplash.com/_WuPjE-MPHo",
+    to: "/rules",
+  },
+  {
+    title: "Entertainment",
+    imgSrc: "https://source.unsplash.com/yubCnXAA3H8",
+    to: "/rules",
+  },
+];
```

Comment:
Maybe you could shift to a dedicated space called `data` or something, So you could also use it for your Home Page. Also, I would have initially used an array for the data structure. But, remember Tanay telling us to use objects instead.